### PR TITLE
Add Motionblinds BLE integration documentation

### PR DESCRIPTION
**Pull request by LennP**
## Proposed change

Add Motionblinds BLE integration documentation.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/109497
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: -

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/motionblinds_ble.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ha_codeowners:
+  - '@LennP'
+  - '@jerrybboy'
+ha_config_flow: true
+ha_platforms:
+  - cover
+ha_integration_type: integration
+---
+
+This integration adds support for [Motionblinds](https://motionblinds.com/) Bluetooth motors. Beware that this integration does not work with *Eve Motionblinds* motors. *Eve Motionblinds* can be added to Home Assistant using the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller/) integration.
```
Comment (c0ffeeca7) [inline source/_integrations/motionblinds_ble.markdown]:
```suggestion
The Motionblinds BLE {% term integration %} adds support for [Motionblinds](https://motionblinds.com/) Bluetooth motors. Beware that this integration does not work with *Eve Motionblinds* motors. *Eve Motionblinds* can be added to Home Assistant using the [HomeKit Device](https://www.home-assistant.io/integrations/homekit_controller/) integration.
```

Comment (c0ffeeca7) [reply source/_integrations/motionblinds_ble.markdown]: `Beware that this integration does not work with *Eve Motionblinds* motors. `
Does this refer to non-Bluetooth motors?

File: source/_integrations/motionblinds_ble.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+{% include integrations/config_flow.md %}
+
+## Setup
+
+During the setup of a Motionblinds BLE device, you will be asked what kind of blind your Motionblind is. There are 8 different blind types:
+
+- **Roller blind**: has the ability to change position and speed.
+- **Honeycomb blind**: has the ability to change position and speed.
+- **Roman blind**: has the ability to change position and speed.
+- **Venetian blind**: has the ability to change position, tilt and speed.
```
Comment (c0ffeeca7) [inline source/_integrations/motionblinds_ble.markdown]:
```suggestion
- **Venetian blind**: has the ability to change position, tilt, and speed.
```

File: source/_integrations/motionblinds_ble.markdown [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+- **Venetian blind (tilt-only)**: has the ability to change tilt and speed.
+- **Double Roller blind**: has the ability to change position, tilt and speed.
```
Comment (c0ffeeca7) [inline source/_integrations/motionblinds_ble.markdown]:
```suggestion
- **Double Roller blind**: has the ability to change position, tilt, and speed.
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @LennP :+1:

File: source/_integrations/motionblinds_ble.markdown (hunk 1 above)
Comment (LennP) [reply source/_integrations/motionblinds_ble.markdown]: Eve Motionblinds do use bluetooth but they are Homekit devices and do not work with this integration

Comment (c0ffeeca7) [reply source/_integrations/motionblinds_ble.markdown]: Thank you for clarifying.

**Pull request merged**